# RXJS-throwError-fromEvent-3

[Edit on StackBlitz ⚡️](https://stackblitz.com/edit/rxjs-catcherror-withmapoperators-7searg)